cura/Settings/MachineManager.py

```diff
@@ -1496,3 +783,268 @@ def setSettingForAllExtruders(self, setting_name: str, property_name: str, prope
         for key, extruder in self._global_container_stack.extruders.items():
             container = extruder.userChanges
             container.setProperty(setting_name, property_name, property_value)
+
+    #
+    # New
+    #
```

Comment:
Remove `# New`. In a few years it won't be new any more.

Reply:
Removed in 632be11